concrete/elements/block_footer_edit.php

```diff
@@ -1,41 +1,36 @@
 <?php
-defined('C5_EXECUTE') or die("Access Denied.");
-?>
+defined('C5_EXECUTE') or die('Access Denied.');
 
-</div>
+/* @var Concrete\Core\Block\Block $b */
 
-<?php
 $bt = $b->getBlockTypeObject();
 
-if (is_array($extraParams)) { // defined within the area/content classes 
-    foreach ($extraParams as $key => $value) {
+?>
+            </div>
+        <?php
+        if (isset($extraParams) && is_array($extraParams)) {
+            // defined within the area/content classes
+            foreach ($extraParams as $key => $value) {
+                ?><input type="hidden" name="<?= $key ?>" value="<?= $value ?>" /><?php
+            }
+        }
+        if (!$b->getProxyBlock() && !$bt->supportsInlineEdit()) {
+            ?>	
+            <div class="ccm-buttons dialog-buttons">
+                <a href="javascript:clickedButton = true;$('#ccm-form-submit-button').get(0).click()" class="btn pull-right btn-primary"><?= t('Save') ?></a>
+                <a style="float:left" href="javascript:void(0)" class="btn btn-default btn-hover-danger" onclick="jQuery.fn.dialog.closeTop()"><?= t('Cancel') ?></a>
+            </div>
+            <?php
+        }
         ?>
-		<input type="hidden" name="<?=$key?>" value="<?=$value?>">
-	<?php 
+        <input type="submit" name="ccm-edit-block-submit" value="submit" style="display: none" id="ccm-form-submit-button" />
+    </form>
+    <?php
+    if ($b->getBlockTypeHandle() === BLOCK_HANDLE_SCRAPBOOK_PROXY) {
```

Comment:
@KorvinSzanto I don't understand to which block of code you are referring to: is it [this one](https://github.com/mlocati/concrete5/blob/de23a54d7ae06268f75c41c90bfec64184ea7ac8/concrete/elements/block_footer_edit.php#L29-L32)?